src/sinks/splunk_hec/logs/integration_tests.rs

```diff
@@ -441,14 +445,16 @@ async fn splunk_auto_extracted_timestamp() {
 
         run_and_assert_sink_compliance(sink, stream::once(ready(event)), &HTTP_SINK_TAGS).await;
 
-        let entry = find_entry(message).await;
+        let entry = find_entry(&message).await;
+
+        // `"2017-10-05T14:33:42.000+00:00
```

Comment:
Perhaps this comment should change to be generic (YYYY-DD-<...>) ?

Reply:
That comment shouldn't have been there.